corehq/apps/smsbillables/migrations/0005_smsbillable_twilio_gateway_fee.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('smsbillables', '0004_bootstrap_smsgh_rates'),
+    ]
+
+    operations = [
```

Comment:
As long as the column is nullable and doesn't have a default value it should be fine. The best way to check is to run this command and view the SQL output:
```
./manage.py sqlmigrate smsbillables 0005
```
This will output the SQL that the migrator will run. If you see anything like `SET DEFAULT` then that's bad.